Fabulous.sln

```diff
@@ -153,6 +153,12 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Fabulous.CodeGen", "Fabulou
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "tests", "tests", "{8B4467AD-8368-4E84-A99F-0354FB746076}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "FabulousWeather", "FabulousWeather", "{4CF4FD79-EA1E-4EB3-A3A9-4D7E38DEFD77}"
+EndProject
+Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "FabulousWeather", "Fabulous.XamarinForms\samples\FabulousWeather\FabulousWeather\FabulousWeather.fsproj", "{39F73F10-B026-4DD4-A1BE-2755C0757C85}"
+EndProject
+Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "FabulousWeather.Droid", "Fabulous.XamarinForms\samples\FabulousWeather\Droid\FabulousWeather.Droid.fsproj", "{F539DD22-3911-47E1-B5E8-810367E80A51}"
```

Comment:
To match what other samples do, could you also provide the other platforms as well?
iOS, macOS, WPF, UWP, GTK

You can do that easily by copy pasting them from another sample (and changing their names, ProjectGuid values, and paths inside the fs/csproj)